gvm/protocols/gmpv7.py

```diff
@@ -3384,6 +3385,7 @@ def get_reports(
                 include note details
             override_details (boolean, optional): If overrides are included,
                 whether to include override details
+            details (boolean, optional): Whether to exclude results
```

Comment:
```suggestion
            details (boolean, optional): Whether to include results
```
As far as I understand, passing `details=True` should result in `details="1"` being set in the XML, shouldn't it?

Reply:
Yes, you are right, the wording is confusing right now. The problem is that **not** passing the option results in getting full details (GMP is inconsistent here compared to other commands).
You have to explicitly set it to `False` in order to suppress them.
So, for consistency, the option should probably be named `no_details` instead:
```suggestion
            no_details (boolean, optional): Whether to exclude results
```

Reply:
I think renaming the parameter to `no_details` makes sense, especially to make it clear that this parameter does not behave like the `details` parameter of the other methods.

Could you update the PR and adapt the tests?

Reply:
Done, with some double negatives unfortunately, but I guess that can't really be avoided in this case due to the GMP design choice here.